### PR TITLE
Add project and about section

### PR DESCRIPTION
**Pull request by Adetunbosun**
I build the middle section of the mobile website  Which include:

The Work section and the "About myself" section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,5 +19,115 @@ <h1 class="title">Michael<br/>Olusola</h1>
...
+    <article class="project-card">
+      <img class="project-image" src="./images/Rectangle-21.png" alt="project image"/>
+      <div class="project-details">
+        <h3 class="project-title">Project name goes here</h3>
+        <ul class="language-list">
+          <li>HTML/CSS</li>
+          <li>Ruby on Rails</li>
+          <li>JavaScript</li>
+        </ul>
+        <button class="project-button" type="submit">See this project &rarr;</button>
```
Comment (Nemwel-Boniface) [inline index.html]: - [ ] You have done an amazing job styling your buttons. There is one small thing left right now. Kindly add the button interaction states for your buttons eg on click, on hover, etc. Currently, your buttons feel very static and are not intuitive. If you may be having doubts of how to achieve this, have a look at [this wonderful resource](https://zellwk.com/blog/style-hover-focus-active-states/).

**The button states from your chosen Figma design:**

image


**Kindly implement this on all the buttons of the project.**

- [ ] Also, kindly note that this type of button might not be a form button. Thus having the type "submit" is not the correct type to add here. I would recommend you change it to a type "button" instead.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Adetunbosun            :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have done a great job with your portfolio project so far :clap: 
You have an excellently done project :fire: perfectly positioned elements and attention to detail :raised_hands: . I will really like to see the finished version of your project :sunglasses: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @Adetunbosun , 

Good job implementing the changes requested by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

5SW


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Adetunbosun) [reply index.html]: Thank you for the review

**Pull request merged**